### ISSUE TITLE
Generated midi file doesn't sound

### ISSUE THREAD

**Issue by bjcancin** [Tonejs/MidiConvert#127]
Hi, I run the following code in node (see [example.zip](https://github.com/Tonejs/Midi/files/5198500/example.zip)) with a midi file generated in musescore 3.5.

```javascript
const { Midi } = require('@tonejs/midi');
const fs = require('fs');

async function run() {

    var midi = new Midi(fs.readFileSync("example.mid"));
    fs.writeFileSync("output.mid", new Buffer(midi.toArray()))

}

run();
```

And the output midi file doesn't sound (see [output.zip](https://github.com/Tonejs/Midi/files/5198503/output.zip)).

Any idea about this problem?

Regards

**Comment by dsholes** [comment]
It seems like somehow the volume and pan aren't being encoded correctly (or at all). The [toArray()](https://github.com/Tonejs/Midi/blob/master/src/Midi.ts#L117) function is calling an [encode](https://github.com/Tonejs/Midi/blob/master/src/Encode.ts#L147) function, so your best bet is to dig through [Encode.js](https://github.com/Tonejs/Midi/blob/master/src/Encode.ts) and try to determine where volume and pan should be written. Unfortunately, I'm not very familiar with MIDI so I'm not sure what part of `encode` needs to be changed.

**Comment by chrisfeltner** [comment]
I have encountered the same issue when trying to play generated MIDI using MediaPlayer in the Android API, but it does sound in AVMIDIPlayer on iOS.

**Comment by chrisfeltner** [comment]
This should be resolved by #134
